$NetBSD$

* Build error:
../document-portal/document-portal-fuse.c:281:37: error: 'FUSE_ROOT_ID' undeclared here
../document-portal/document-portal-fuse.c:395:15: error: 'FUSE_SET_ATTR_MODE' undeclared
../document-portal/document-portal-fuse.c:398:15: error: 'FUSE_SET_ATTR_UID' undeclared
...

--- document-portal/meson.build.orig	2026-01-12 14:49:05.731913861 +0000
+++ document-portal/meson.build
@@ -59,14 +59,14 @@ xdg_document_portal_sources = [
   permission_store_built_sources,
 ]
 
-xdg_document_portal = executable(
-  'xdg-document-portal',
-  xdg_document_portal_sources,
-  dependencies: [xdg_desktop_portal_deps, fuse3_dep],
-  include_directories: incs_xdg_desktop_portal,
-  install: true,
-  install_dir: libexecdir,
-)
+#xdg_document_portal = executable(
+#  'xdg-document-portal',
+#  xdg_document_portal_sources,
+#  dependencies: [xdg_desktop_portal_deps, fuse3_dep],
+#  include_directories: incs_xdg_desktop_portal,
+#  install: true,
+#  install_dir: libexecdir,
+#)
 
 configure_file(
   input: 'xdg-document-portal.service.in',
@@ -93,4 +93,4 @@ permission_portal_service_file = configu
   configuration: base_config,
   install: true,
   install_dir: dbus_service_dir,
-)
\ No newline at end of file
+)
